src/libcddb-1-fixes.patch

```diff
@@ -0,0 +1,73 @@
+This file is part of MXE. See LICENSE.md for licensing information.
+
+Contains ad hoc patches for cross building.
+
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Norbert Schlia <nschlia@oblivion-software.de>
+Date: Mon, 12 Dec 2016 08:48:17 +0100
+Subject: [PATCH] Newer versions winioctl.h contain a #define SEARCH_ALL ..
+ which collides with the enum in cddb_conn.h.
+
+
+diff --git a/include/cddb/cddb_conn.h b/include/cddb/cddb_conn.h
+index 1111111..2222222 100644
+--- a/include/cddb/cddb_conn.h
++++ b/include/cddb/cddb_conn.h
+@@ -59,6 +59,10 @@ typedef struct cddb_conn_s cddb_conn_t;
+  * Which fields to use for the full text search is defined by one or
+  * more of the constants below.
+  */
++#ifdef SEARCH_ALL
++#undef SEARCH_ALL   // Defined in winioctl.h, causes havoc under windoze
++#endif
```

Comment:
(Not in this PR.)
Can you fix / report this to upstream, please? If this symbol is renamed to SEARCH_EVERYTHING (or other name) in next release of libcddb, we could remove this patch.